### PR TITLE
Production setup, remove leftovers of ITS/MFT strobe lengths and thro…

### PR DESCRIPTION
**Pull request by catalinristea**
…ttling setting for grid

Ciao @chiarazampolli and @noferini,
Please review/amend/merge if you find these useful.
Thanks, Catalin.

### CODE REVIEW

File: DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh [hunk 1]
```diff
@@ -80,7 +80,7 @@ export ARGS_EXTRA_PROCESS_o2_tof_reco_workflow="--use-ccdb --for-calib"
 export PVERTEXER="pvertexer.acceptableScale2=9;pvertexer.minScale2=2.;pvertexer.nSigmaTimeTrack=4.;pvertexer.timeMarginTrackTime=0.5;pvertexer.timeMarginVertexTime=7.;pvertexer.nSigmaTimeCut=10;pvertexer.dbscanMaxDist2=36;pvertexer.dcaTolerance=3.;pvertexer.pullIniCut=100;pvertexer.addZSigma2=0.1;pvertexer.tukey=20.;pvertexer.addZSigma2Debris=0.01;pvertexer.addTimeSigma2Debris=1.;pvertexer.maxChi2Mean=30;pvertexer.timeMarginReattach=3.;pvertexer.addTimeSigma2Debris=1.;pvertexer.dbscanDeltaT=24;pvertexer.maxChi2TZDebris=100;pvertexer.maxMultRatDebris=1.;pvertexer.dbscanAdaptCoef=20.;pvertexer.timeMarginVertexTime=1.3"
 export SVTX="svertexer.checkV0Hypothesis=false;svertexer.checkCascadeHypothesis=false"
 
-export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT;$PVERTEXER;ITSAlpideParam.roFrameLengthInBC=$ITS_STROBE;MFTAlpideParam.roFrameLengthInBC=$MFT_STROBE;"
+export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT"
```
Comment (chiarazampolli) [inline DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh:83]: You removed also $PVERTEXER, it should be left. Can you check? For the other file it is correct. Thanks!

**Comment by catalinristea** [comment]
Ciao @chiarazampolli,
Thanks for noticing it, now it should be fine.
Yours, Catalin.

File: DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh [hunk 2]
```diff
@@ -80,7 +80,7 @@ export ARGS_EXTRA_PROCESS_o2_tof_reco_workflow="--use-ccdb --for-calib"
 export PVERTEXER="pvertexer.acceptableScale2=9;pvertexer.minScale2=2.;pvertexer.nSigmaTimeTrack=4.;pvertexer.timeMarginTrackTime=0.5;pvertexer.timeMarginVertexTime=7.;pvertexer.nSigmaTimeCut=10;pvertexer.dbscanMaxDist2=36;pvertexer.dcaTolerance=3.;pvertexer.pullIniCut=100;pvertexer.addZSigma2=0.1;pvertexer.tukey=20.;pvertexer.addZSigma2Debris=0.01;pvertexer.addTimeSigma2Debris=1.;pvertexer.maxChi2Mean=30;pvertexer.timeMarginReattach=3.;pvertexer.addTimeSigma2Debris=1.;pvertexer.dbscanDeltaT=24;pvertexer.maxChi2TZDebris=100;pvertexer.maxMultRatDebris=1.;pvertexer.dbscanAdaptCoef=20.;pvertexer.timeMarginVertexTime=1.3"
 export SVTX="svertexer.checkV0Hypothesis=false;svertexer.checkCascadeHypothesis=false"
 
-export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT;$PVERTEXER;ITSAlpideParam.roFrameLengthInBC=$ITS_STROBE;MFTAlpideParam.roFrameLengthInBC=$MFT_STROBE;"
+export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT;$PVERTEXER"
```
Comment (chiarazampolli) [inline DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh:83]: Here I think it is better to leave the ";" at the end. Sorry for not noticing before.

File: DATA/production/configurations/2022/LHC22f/apass1_TPCcalib/setenv_extra.sh [hunk 3]
```diff
@@ -80,7 +80,7 @@ export ARGS_EXTRA_PROCESS_o2_tof_reco_workflow="--use-ccdb --for-calib"
 export PVERTEXER="pvertexer.acceptableScale2=9;pvertexer.minScale2=2.;pvertexer.nSigmaTimeTrack=4.;pvertexer.timeMarginTrackTime=0.5;pvertexer.timeMarginVertexTime=7.;pvertexer.nSigmaTimeCut=10;pvertexer.dbscanMaxDist2=36;pvertexer.dcaTolerance=3.;pvertexer.pullIniCut=100;pvertexer.addZSigma2=0.1;pvertexer.tukey=20.;pvertexer.addZSigma2Debris=0.01;pvertexer.addTimeSigma2Debris=1.;pvertexer.maxChi2Mean=30;pvertexer.timeMarginReattach=3.;pvertexer.addTimeSigma2Debris=1.;pvertexer.dbscanDeltaT=24;pvertexer.maxChi2TZDebris=100;pvertexer.maxMultRatDebris=1.;pvertexer.dbscanAdaptCoef=20.;pvertexer.timeMarginVertexTime=1.3"
 export SVTX="svertexer.checkV0Hypothesis=false;svertexer.checkCascadeHypothesis=false"
 
-export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT;$PVERTEXER;ITSAlpideParam.roFrameLengthInBC=$ITS_STROBE;MFTAlpideParam.roFrameLengthInBC=$MFT_STROBE;"
+export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="TPCGasParam.DriftV=$VDRIFT;$PVERTEXER"
```
Comment (chiarazampolli) [inline DATA/production/configurations/2022/LHC22f/apass1_TPCcalib/setenv_extra.sh:83]: Same as above, please keep the ";"

**Comment by chiarazampolli** [comment]
Thanks @catalinristea , just two more minor comments.

**Comment by chiarazampolli** [comment]
> Ciao @chiarazampolli,
> 
> Done, thanks! Other options just below this one (CONFIG_EXTRA_PROCESS_o2_tpcits_match_workflow, CONFIG_EXTRA_PROCESS_o2_tof_matcher_workflow, CONFIG_EXTRA_PROCESS_o2_trd_global_tracking) were not having this ending semicolon, so I thought also not to add it. For sure it wouldn't hurt :)
> 
> Yours, Catalin.

Thanks @catalinristea ! Maybe we should add the ";" everywhere, but let's not do it now. 

Chiara

**Review by chiarazampolli — APPROVED** [review]

**Pull request merged**